### PR TITLE
Add a check to wait and validate HPP status 

### PR DESCRIPTION
**Pull request by geetikakay**
Wait for HostPathProvisioner  to be available

### CODE REVIEW

**Comment by geetikakay** [comment]
@newkit @sean-freeman Hello, could you please help in reviewing this patch

File: roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-hpp.yml [hunk 1]
```diff
@@ -71,6 +71,22 @@
           nodeSelector:
             kubernetes.io/os: linux
 
+- name: Wait for HostPathProvisioner status to become Available
+  kubernetes.core.k8s_info:
+    api_version: hostpathprovisioner.kubevirt.io/v1beta1
+    kind: HostPathProvisioner
+    name: hostpath-provisioner
+  register: hostpath_status
+  until: >
```
Comment (0xFelix) [inline roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-hpp.yml:80]: Would it be possible to use `wait_condition` instead?

https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_info_module.html#parameter-wait_condition

Comment (geetikakay) [reply roles/sap_hypervisor_node_preconfigure/tasks/platform/redhat_ocp_virt/install-hpp.yml]: added and tested

**Review by 0xFelix — APPROVED** [review]
/lgtm

**Review by newkit — APPROVED** [review]
lgtm2

**Pull request merged**